components/header/__snapshots__/index.test.tsx.snap

```diff
@@ -861,10 +861,28 @@ font-display: block;",
               />
             </a>
           </Link>
-          <div
-            className="PodBrowser-header-banner__main-nav"
-          />
           <UserMenu>
+            <div
+              className="PodBrowser-header-banner__main-nav"
+            >
+              <Link
+                href="/contacts"
+                replace={true}
+              >
+                <button
```

Comment:
Do we have styles for button-style links? I think the issue here is that we want to prefer using the next `Link` component instead of using the router directly. I used the prism components that matched the design so I ended up with a button and a link.

Reply:
There's the LinkButton component, it's a variant of the Button component (can be imported from the same location).

Reply:
(Really need to get Prism/Storybook to show the React Components that are available through Prism...)

Reply:
`LinkButton` is not the right button for use in this header menu. It looks more like a button than the header navigation links.